javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -333,23 +333,25 @@ protected void buildUnexpectedResponseExceptionTypes(ProxyMethod.Builder builder
             swaggerExceptionDefinitions.exceptionTypeMapping);
         mergedExceptionTypeMapping.putAll(settingsExceptionTypeMap);
 
-        // Convert the exception type mapping into what code generation uses elsewhere.
-        Map<ClassType, List<Integer>> processedMapping = new HashMap<>();
-        for (Map.Entry<Integer, ClassType> kvp : mergedExceptionTypeMapping.entrySet()) {
-            processedMapping.compute(kvp.getValue(), (errorType, statuses) -> {
-                if (statuses == null) {
-                    List<Integer> statusList = new ArrayList<>();
-                    statusList.add(kvp.getKey());
-                    return statusList;
-                }
+        if (!settings.isLowLevelClient()) {
+            // Convert the exception type mapping into what code generation uses elsewhere.
+            Map<ClassType, List<Integer>> processedMapping = new HashMap<>();
+            for (Map.Entry<Integer, ClassType> kvp : mergedExceptionTypeMapping.entrySet()) {
+                processedMapping.compute(kvp.getValue(), (errorType, statuses) -> {
+                    if (statuses == null) {
+                        List<Integer> statusList = new ArrayList<>();
+                        statusList.add(kvp.getKey());
+                        return statusList;
+                    }
 
-                statuses.add(kvp.getKey());
-                return statuses;
-            });
-        }
+                    statuses.add(kvp.getKey());
+                    return statuses;
+                });
+            }
 
-        if (!processedMapping.isEmpty()) {
-            builder.unexpectedResponseExceptionTypes(processedMapping);
+            if (!processedMapping.isEmpty()) {
+                builder.unexpectedResponseExceptionTypes(processedMapping);
```

Comment:
Skip list of unexpected response exception type for LLC, as it does not have models for exception class.